in_toto/runlib.py

```diff
@@ -528,7 +542,7 @@ def in_toto_run(name, material_list, product_list, link_cmd_args,
 
   if link_cmd_args:
     LOG.info("Running command '{}'...".format(" ".join(link_cmd_args)))
-    byproducts = execute_link(link_cmd_args, record_streams)
+    byproducts = execute_link(link_cmd_args, record_streams, quiet=True)
```

Comment:
```suggestion
    byproducts = execute_link(link_cmd_args, record_streams, quiet)
```

Reply:
By using `quiet=True`, you're not actually using the `quiet` that was passed to `in_toto_run`, but rather setting a keyword arg for `execute_link`. :)